### PR TITLE
hydra

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: services/hydra/default.nix [hunk 1]
```diff
@@ -33,18 +28,6 @@ in
       "https://github.com/NixOS/"
     ];
 
-    nixpkgs.config = {
```
Comment (zowoq) [inline services/hydra/default.nix:36]: @zimbatm nixpkgs.config was added in 19581331ed38654514ee14f09a5c19123da7da8e, I don't think we need it now?

Comment (zimbatm) [reply services/hydra/default.nix:36]: I don't know if those still apply now that we use `nixpkgs.lib.nixosSystem`? I think it's using the passed instance of nixpkgs instead.

Comment (zowoq) [reply services/hydra/default.nix:36]: Sorry, I meant that I don't think that we're still running builds for anything that requires them?

Comment (zimbatm) [reply services/hydra/default.nix:36]: I propose merging this and find out :)

**Pull request merged**